po/es.po

```diff
@@ -0,0 +1,1212 @@
+msgid ""
+"The [Ferrocene Language Specification](https://spec.ferrocene.dev) is "
+"MIT/Apache-2.0"
+msgstr ""
+"La [Especificación del Lenguaje de Ferrocene](https://spec.ferrocene.dev) es "
+"MIT/Apache-2.0"
+
+#: src\support.md:7
+msgid ""
+"You can even [read the Qualification "
+"Documents](https://public-docs.ferrocene.dev/main/index.html) right now"
+msgstr ""
+"Hasta puedes [leer los Documentos de "
+"Calificación](https://public-docs.ferrocene.dev/main/index.html) ahorita"
+
+#: src\support.md:12
+msgid "And you sell?"
+msgstr "¿Y qué vendes"
+
+#: src\support.md:14
+msgid "Binaries"
+msgstr "Binarios"
+
+#: src\support.md:15
+msgid "Long-term Support"
+msgstr "Soporte de largo plazo"
+
+#: src\support.md:16
+msgid "for Ferrocene"
+msgstr "para Ferrocene"
+
+#: src\support.md:17
+msgid "for third-party crates (as an add-on)"
+msgstr "y para crates de terceros (por aparte)"
+
+#: src\support.md:18
+msgid "Includes digitally-signed Qualification Documents for your assessor"
+msgstr "Incluye los Documentos de Calificación firmados digitalmente para tu asesor"
+
+#: src\support.md:20
+msgid "How long is long-term?"
+msgstr "¿Qué tan largo es largo plazo?"
+
+#: src\support.md:22
+msgid "Upstream Rust only supports each release for six weeks"
+msgstr "Upstream Rust solo da soporte a cada release por 6 semanas"
+
+#: src\support.md:23
+msgid "There are no backports to old stable releases"
+msgstr "No hay backports a releases viejos y estables"
+
+#: src\support.md:24
+msgid "We pick a _rolling_ release, _stabilise_ it, and support it for 2 years"
+msgstr "Escogemos un _rolling_ release, lo _estabilizamos_ y le damos soporte por 2 años"
+
+#: src\support.md:25
+msgid "We give subscribers bug-fix releases as we fix bugs"
+msgstr "Le damos a los suscriptores releases de bug-fixes mientras los encontramos"
+
+#: src\support.md:26
+msgid "After 2 years it goes into Extended Support"
+msgstr "Después de 2 años entra a Soporte Extendido"
+
+#: src\support.md:27
+msgid "We tell subscribers about issues and give possible workarounds"
+msgstr "Le decimos a nuestros suscriptores sobre issues y posibles mitigaciones"
+
+#: src\support.md:29
+msgid "Roadmap"
+msgstr "Plan a futuro"
+
+#: src\support.md:31
+msgid "![Ferrocene Release Model](./images/rolling.png)"
+msgstr ""
+
+#: src\support.md:33
+msgid "Terminology"
+msgstr "Terminología"
+
+#: src\support.md:35
+msgid "What upstream calls _stable_, Ferrocene renames to _rolling_."
+msgstr ""
+
+#: src\support.md:36
+msgid "What upstream calls _beta_, Ferrocene renames _pre-rolling_."
+msgstr ""
+
+#: src\support.md:37
+msgid "Ferrocene has its own _beta_ channel, cut from _rolling_."
+msgstr ""
+
+#: src\support.md:38
+msgid "Ferrocene has its own _stable_ channel, cut from _beta_."
+msgstr ""
+
+#: src\support.md:40
+msgid "How much is it?"
+msgstr "¿Cuánto cuesta?"
+
+#: src\support.md:42
+msgid "Access to the open-source code is free"
+msgstr "Accesar el código open source es gratis"
```

Comment:
I'd say "Acceder al" instead of "Accesar el"

Reply:
Perfecto.